apps/site/src/views/Schedule/sections/ClipboardSchedule/ClipboardSchedule.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import Image from "next/image";
+
+import Accordion from "react-bootstrap/Accordion";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+
+import clip from "@/assets/images/clip.svg";
+
+import styles from "./ClipboardSchedule.module.scss";
+
+interface ClipboardScheduleProps {
+	schedule: {
+		_key: string;
+		title: string;
+		description: string;
+		location: string;
+		startDate: Date;
+		endDate: Date;
+	}[];
+}
+
+function ClipboardSchedule({ schedule }: ClipboardScheduleProps) {
+	return (
+		<Container
+			as="section"
+			className={styles.clipboard + " px-0 position-relative"}
+		>
+			<div className={styles.clip}>
+				<Image src={clip} alt="Clipboard clip" className={styles.clip} />
+			</div>
+			<h2 className="mb-5">Countdown Timer</h2>
+			<Accordion defaultActiveKey="0" className={styles.accordion}>
+				{schedule.map(
+					(
+						{ _key, title, description, location, startDate, endDate },
+						index,
+					) => (
+						<Accordion.Item
+							key={_key}
+							eventKey={`${index}`}
+							className={styles.accordionItem}
+						>
+							<Accordion.Header as="div" className={styles.accordionHeader}>
+								<Row className="w-100 justify-content-between align-items-center">
+									<Col sm="auto">
+										<h3 className="text-start">{title}</h3>
```

Comment:
Note [non-blocking]: putting a heading inside a button (which is what `Accordion.Header` provides) removes the heading semantics (no longer document accessible). If we made the entire `Accordion.Header` as `h3`, then the location/time would also be included in the heading. Unsure what the best way to handle this is.

Reply:
I simplified it a little by having be a button like how it is by default and then putting the h3 and span next to each other. I'm not too sure if there's a way to resolve this unless we build it ourselves.
```
<Accordion.Header className={styles.accordionHeader}>
	<h3>{title}</h3>
	<span className="text-end ms-auto">{location}, 1pm-2pm</span>
</Accordion.Header>
```

Reply:
The problem is the browser doesn't interpret the `h3` as an actual heading since it's inside a button.